### DISCUSSION TITLE
Lack of ad Impressions for rewarded video ad

### DISCUSSION THREAD

**Discussion by ponarangan** [airsdk/Adobe-Runtime-Support#3571] [Q&A]
My app gets a very few rewarded video ad impressions compared to the ad requests it receives. For certain geographies no ad impression at all for the ad requests. Match rate is 100% but it is getting very few ad impression ( For 20 ad requests , 1 or 2 ad impressions). What should I do to get 100%  ad impressions for the ad requests I get. Kindly help.

**Comment by mrfrasier** [comment]
Are you using an ANE to integrate an SDK? If so, the maker of the ANE would probably have better insight

**Comment by ponarangan** [comment]
We are using air native extensions

**Comment by mrfrasier** [comment]
> We are using air native extensions

Sure, but WHICH? Are you using the Ironsource extension by Distriqt? If so, @marchbold can probably help. If not, which extension? 

In reality, this isn't an AIR SDK issue as the AIR SDK does not do this natively. This is an issue for the maker of the ANE you are using, so the more information you provide, the easier it should be to point you toward help

**Comment by marchbold** [comment]
Hi, If you are having an issue with one of our extensions please use the repository associated with it to log an issue and we'll help you through it. 

Advertising is likely either: 
- https://github.com/distriqt/ANE-IronSource/
- https://github.com/distriqt/ANE-Adverts/
